brokers/theia/broker.go

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2012-2018 Red Hat, Inc.
+// Copyright (c) 2019 Red Hat, Inc.
```

Comment:
Should not license contain first contribution year and last modification year? Like `2018-2019`?

Reply:
:+1: